Lib/fontTools/pens/ttGlyphPen.py

```diff
@@ -47,36 +54,136 @@ def __init__(self, glyphSet, handleOverflowingTransforms=True):
         self.handleOverflowingTransforms = handleOverflowingTransforms
         self.init()
 
-    def init(self):
+    def _isClosed(self):
+        """
+        Check if the current path is closed.
+        """
+        raise NotImplementedError
+
+    def init(self) -> None:
         self.points = []
         self.endPts = []
         self.types = []
         self.components = []
+        self._currentContourStartIndex = None
```

Comment:
I currently don't see this used as an index, just as a flag meaning "we're in between beginPath and endPath".

Reply:
I moved it to the subclass.